azure-devops/projects/pagopa-projects/tslcert-prod-config-prod-plaform-pagopa-it.tf

```
@@ -0,0 +1,74 @@
+variable "tlscert-prod-config-prod-platform-pagopa-it" {
+  default = {
+    repository = {
+      organization   = "pagopa"
+      name           = "le-azure-acme-tiny"
+      branch_name    = "master"
+      pipelines_path = "."
+    }
+    pipeline = {
+      enable_tls_cert         = true
+      path                    = "TLS-Certificates\\PROD"
+      dns_record_name         = ""
+      dns_zone_name           = "config.platform.pagopa.it"
```

```suggestion
      dns_record_name         = "config"
      dns_zone_name           = "platform.pagopa.it"
```
dns zone is `platform.pagopa.it` and the record `config`